docs/ops/arithmetic/Add_1.md

```diff
@@ -6,46 +6,46 @@
 
 **Short description**: *Add* performs element-wise addition operation with two given tensors applying multi-directional broadcast rules.
 
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
+
+After broadcasting *Add* performs addition operation for the input tensors *a* and *b* using formula below:
```

Comment:
`using formula below` -> `using the formula below`

Reply:
Corrected.